### DISCUSSION TITLE
Welcome to the direnv community

### DISCUSSION THREAD

**Discussion by zimbatm** [direnv/direnv#1074] [General]
`direnv` was a tool I built for myself long ago. I was happily using it for my ruby development at the time. And then it started growing. I remember being surprised when I joined a company and discovered they were using direnv. It was also a fun moment when one employee asked me if I knew about that project :-)

The point is that direnv has grown and is not about myself anymore. People and companies are using it. It's time to open up the discussions and allow more people to contribute to the project.

So welcome. This is the place where we'll have conversations and get organized.

**Comment by gluxon** [comment]
> I remember being surprised when I joined a company and discovered they were using direnv. It was also a fun moment when one employee asked me if I knew about that project :-)

That's fantastic and hilarious! Congrats for creating such a useful tool. I use it every day and love it.
